web/profiles/joinup/joinup.behat.inc

```diff
@@ -0,0 +1,163 @@
+<?php
+
+/**
+ * @file
+ * Contains \UserSubContext.
```

Comment:
Maybe this class should move to a different file or be renamed to JoinupSubContext in order to give the opportunity to add code for more core functionalities such as User entity.